features/diffing.feature

```diff
@@ -51,13 +73,59 @@ Feature: diffing
     Then the output should contain:
       """
              Diff:
-             @@ -1,2 +1,4 @@
+             @@ -1,2 +1,5 @@
              -/expected/m
              +this is the
              +  actual
              +    string
       """
 
+  Scenario: diff for a multiline string and a regexp
```

Comment:
This seems to be a duplicate of another scenario, was this committed accidentally?